cloud.profiles.d/edx_sandbox.conf

```diff
@@ -0,0 +1,31 @@
+# This profile is for instances launched for the purpose of
+# building edX sandbox AMIs.
+
+edx_sandbox_ami:
+  provider: mitx
+  size: m3.medium
+  image: ami-427a392a
+  ssh_username: ubuntu
+  ssh_interface: public_ips
+  securitygroupid: sg-1d437d65
+  subnetid: subnet-8671faac
+  block_device_mapping:
+    - DeviceName: /dev/xvda
+      Ebs.VolumeSize: 50
```

Comment:
Is this the volume size that we are currently using for EdX Instances? Can we choose a different volume size when launching an instance from the generated AMI?